stdlib/public/core/Bool.swift

```diff
@@ -146,6 +146,18 @@ extension Bool : Equatable, Hashable {
   }
 }
 
+extension Bool : LosslessStringConvertible {
```

Comment:
Please make sure we have tests for this.  It would be best if you could add a utility to `StdlibUnittest` that could test any `LosslessStringConvertible` and `Equatable` type, by accepting a list of instances.